src/Service/CloudFormation/src/ValueObject/RollbackConfiguration.php

```diff
@@ -23,7 +23,7 @@ final class RollbackConfiguration
      */
     public function __construct(array $input)
     {
-        $this->RollbackTriggers = array_map([RollbackTrigger::class, 'create'], $input['RollbackTriggers'] ?? []);
+        $this->RollbackTriggers = isset($input['RollbackTriggers']) ? array_map([RollbackTrigger::class, 'create'], $input['RollbackTriggers']) : null;
```

Comment:
Hm.. 

```php
$x = new RollbackConfiguration();
$x->getRollbackTriggers(); // will throw type error
```

Reply:
indeed. We should either change the return typehint of getter (BC break) 
Or fallback to empty array